ffka-gluon-web-usb-wan-hotplug/Makefile

```diff
@@ -11,4 +11,9 @@ define Package/ffka-gluon-web-usb-wan-hotplug
   DEPENDS:=+gluon-web-admin +ffda-usb-wan-hotplug
 endef
 
+define Package/ffka-gluon-web-usb-wan-hotplug
```

Comment:
This is an invalid redefinition. Also, if you'd like to provide a package description, adding a Markdown file in the packages root is preferred, as the description in the Makefile does not provide styling.

Reply:
oh, you are right, i forgot `/description` at the end.

But I created a Readme instead now.